frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -307,84 +339,117 @@ function roleSelected(evt: any) {
+                                    ></InputText>
+                                </Field>
+                                <small
+                                    class="helper-text"
+                                    v-if="
+                                        !forestClientInactive &&
+                                        !forestClientDuplicate
                                     "
-                                    :validateOnChange="true"
-                                    v-bind="field"
-                                    class="w-100"
-                                    v-on:keypress="
-                                        forestClientCheckOnlyDigit($event)
+                                    >Add and verify the Client IDs. Add multiple
+                                    numbers by separating them with
+                                    commas</small
+                                >
+                                <small
+                                    class="invalid-feedback"
+                                    v-if="
+                                        forestClientInactive ||
+                                        forestClientDuplicate
                                     "
-                                    @input="resetForestClientNumberData()"
-                                    :class="{
-                                        'is-invalid': errors.forestClientNumber,
-                                    }"
-                                ></InputText>
-                            </Field>
-                            <ErrorMessage
-                                class="invalid-feedback"
-                                name="forestClientNumber"
-                            />
-                        </div>
-
-                        <div class="col-md-2">
-                            <Button
-                                class="button"
-                                outlined
-                                aria-label="Verify forest client number"
-                                :name="'verifyFC'"
-                                :label="'Verify'"
-                                :loading-label="'Verifying...'"
-                                @click="
-                                    verifyForestClientNumber(
-                                        formData.forestClientNumber as string
-                                    )
-                                "
-                                v-bind:disabled="
-                                    formData.forestClientNumber?.length < 8 ||
-                                    LoadingState.isLoading.value
-                                "
-                            >
-                            </Button>
+                                >
+                                    {{
+                                        `Client ID ${
+                                            formData.forestClientNumber
```

Comment:
I assume we should be able to see the the forest client number which is inactive or duplicate, but seems like this data is none, it didn't show as expected
<img width="358" alt="Screen Shot 2023-10-11 at 1 48 55 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/f9d6d320-f1c6-4f44-9c43-c674f83c80c2">